plugin.audio.talandaviad/addon.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.audio.talandaviad" name="Tal &amp; Aviad" version="0.0.1" provider-name="Nir Elbaz">
+    <requires>
+        <import addon="xbmc.python" version="2.24.0"/>
+        <import addon="script.module.beautifulsoup4" version="4.6.0"/>
+        <import addon="script.module.requests" version="2.18.1"/>
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="addon.py">
+        <provides>audio</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="he">תוכנית הבוקר של טל ואביעד ב- 99FM</summary>
+        <summary lang="en">Tal &amp; Aviad morning show at 99 FM</summary>
+        <description lang="he">
+            האזינו לכל ההקלטות מתוכנית הבוקר עם טל ברמן ואביעד קיסוס - תוכנית הבוקר המואזנת ביותר. כל בוקר ב- 7:00
+        </description>
+        <description lang="en">
+            VOD of the morning show with Tal &amp; Aviad, the most popular radio program in Israel, every day at 07:00
+        </description>
+        <disclaimer lang="en">This is an official ECO 99FM Kodi add-on</disclaimer>
+        <language>he</language>
+        <platform>all</platform>
+        <license>GNU GENERAL PUBLIC LICENSE Version 2, June 1991</license>
+        <website>http://eco99fm.maariv.co.il/</website>
+        <email>nire0510@gmail.com</email>
+        <source>https://github.com/nire0510/plugin.audio.talandaviad</source>
+        <assets>
+            <icon>icon.png</icon>
```

Comment:
Because someone wrote a tutorial 10 years ago and it just stuck around? Who knows? Those are in no way rules though. (it even says 'all the above is a recommended outline')